Doc/library/csv.rst

```diff
@@ -492,9 +492,9 @@ DictReader objects have the following public attribute:
 Writer Objects
 --------------
 
-:class:`Writer` objects (:class:`DictWriter` instances and objects returned by
+:class:`Writer<writer>` objects (:class:`DictWriter` instances and objects returned by
```

Comment:
We usually add a space here:
```suggestion
:class:`Writer <writer>` objects (:class:`DictWriter` instances and objects returned by
```

Although, I don't see a `Writer` class with capital W, is it only capitalised because it begins the sentence? If so, I'd either reword so it's not first, or leave it lowercase.

Reply:
Yes, I left it capitalized. I suspect the original author may have done it that way. I just added a proper reference. The `csv` module dates from the days when types (written in C) and classes (written in Python) were not quite the same thing. Type names generally weren't capitalized (I think, hence `csv.writer`), while classes were, hence `csv.DictWriter`.